src/ios/FabricPlugin.m

```diff
@@ -246,6 +247,11 @@ - (void)sendCrash:(CDVInvokedUrlCommand*)command
     [[Crashlytics sharedInstance] crash];
 }
 
+- (void)recordError:(CDVInvokedUrlCommand*)command
+{
+    [[Crashlytics sharedInstance] recordError];
```

Comment:
This is a compilation error; the `recordError` selector takes an `NSError` object.

Also, the `message` parameter is not being passed here.